lib/sparql_to_sw_solr/physical_field.rb

```diff
@@ -0,0 +1,51 @@
+module SparqlToSwSolr
+  class InstanceSolrDoc
+    module PhysicalField
+
+      private
+
+      def physical_values
+        physical_solutions.each_solution do |soln|
+          next unless soln.bindings.keys.include?(:noteType)
+          @note_type = soln.noteType.to_s
+          @physical1 = soln.extentLabel_1.to_s
+          @physical3 = soln.instanceDimensions_3.to_s
+          note_case(soln)
+        end
+        physical_details
+      end
+
+      def note_case(soln)
+        case @note_type
+        when 'Physical details'
+          @physical2 = soln.noteLabel.to_s
+        when 'Accompanying materials'
+          @physical4 = soln.noteLabel.to_s
+        end
+      end
```

Comment:
I _think_ there is a problem with using instance variables in this module.  I _think_ there will be hang-over effects in the iterations - there will be iterations where the `@physical2` or the `@physical4` should be nil or blank, but they will not be because of a prior iteration that sets the value and it persists across iterations.  I'm not clear on the answer to this yet, I think we might need to pair on it rather than comment extensively here.